samples/powershell/AddSharePointMappings.ps1

```diff
@@ -1,16 +1,20 @@
 Try {
-    [System.Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+    [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12
```

Comment:
The fly server is on TLS1.3 now。

Reply:
This is for compatibility with lower versions of the system which default protocol type is not Tls 1.2, for Tls 1.3 we will do more research.